ld/zld.xcodeproj/project.pbxproj

```diff
@@ -1308,6 +1308,7 @@
 				SECTORDER_FLAGS = "";
 				STRIP_INSTALLED_PRODUCT = NO;
 				TOOLCHAINS = osx;
+				VALID_ARCHS = x86_64;
```

Comment:
why set it to just x86_64?

Reply:
This is for a dependency of the tests, it doesn't cross compile. Haven't figured out why.

Reply:
Tried again and it passes now, reverting.